_integration-test/test_run.py

```diff
@@ -171,3 +182,64 @@ def placeholder_fn():
         client,
         lambda x: len(json.loads(x)["data"]) > 0,
     )
+
+
+def test_customizations():
+    commands = [
+        [
+            "docker",
+            "compose",
+            "--ansi",
+            "never",
+            "run",
+            "--no-deps",
+            "web",
+            "bash",
+            "-c",
+            "if [ ! -e /created-by-enhance-image ]; then exit 1; fi",
+        ],
+        [
+            "docker",
+            "compose",
+            "--ansi",
+            "never",
+            "run",
+            "--no-deps",
+            "--entrypoint=/etc/sentry/entrypoint.sh",
+            "sentry-cleanup",
+            "bash",
+            "-c",
+            "if [ ! -e /created-by-enhance-image ]; then exit 1; fi",
+        ],
+        [
+            "docker",
+            "compose",
+            "--ansi",
+            "never",
+            "run",
+            "--no-deps",
+            "web",
+            "python",
+            "-c",
+            "import ldap",
+        ],
+        [
+            "docker",
+            "compose",
+            "--ansi",
+            "never",
+            "run",
+            "--no-deps",
+            "--entrypoint=/etc/sentry/entrypoint.sh",
+            "sentry-cleanup",
+            "python",
+            "-c",
+            "import ldap",
+        ]
+    ]
+    for command in commands:
+        result = subprocess.run(command, check=False)
+        if os.getenv("TEST_CUSTOMIZATIONS", "False") == "True":
+            assert result.returncode == 0
```

Comment:
Does `test_customization` run in both matrix variants?

Reply:
Yes, it does. If customizations enabled, it checks to ensure that they are truly enabled by making sure a dependency can be imported that is included in the custom image (ldap). If not, it checks to ensure that an error is thrown when trying to import a dependency that comes from the custom image.